CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Escape sequence to save and restore window title from stack
 - Alternate scroll escape sequence (`CSI ? 1007 h` / `CSI ? 1007 l`)
 - Print name of launch command if Alacritty failed to execute it
+- Track and report damage to supported compositors (primarily Wayland)
```

Comment:
to supported -> to compatible ? What do you think though?

Reply:
I have no stronf opinion, but I usually use "support" when it comes to optional features, e.g. `swap_buffers_with_damage_supported`.

Compatible has a tone more along the lines of backward or alternate technology compatibility, e.g. "compatible with Windows applications".

Reply:
I mean, that supported usually means "supported by us", however compatible means "system has all the requirements(supports damage) for us to do something", but its just up to you, and I'm probably wrong here.